backend/tests/integration-async/alembic/test_alembic_data_migrations.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import datetime
+
+from flask_alembic import Alembic
+from invenio_db import db
+from sqlalchemy.dialects.postgresql import UUID
+
+
+def sql_insert_to_pidstore_pid(data, db):
+    db.session.execute(
+        """
+        INSERT INTO public.pidstore_pid(
+        created, updated, id, pid_type, pid_value,
+        pid_provider, status, object_type, object_uuid
+        ) VALUES ('{0}','{1}', {2}, '{3}', '{4}','{5}', '{6}', '{7}', '{8}');
+        """.format(
+            *data
+        )
+    )
```

Comment:
this is what `sqlalchemy.text` is for: https://docs.sqlalchemy.org/en/13/core/tutorial.html#using-text-fragments-inside-bigger-statements.